### PR TITLE
Updating Vungle to 6.10.3

### PR DESCRIPTION
**Pull request by jiancchen**

### CODE REVIEW

**Review by palrahul — APPROVED** [review]
lgtm

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java [hunk 1]
```diff
@@ -248,6 +248,10 @@ public void requestBannerAd(@NonNull Context context,
       }
       return;
     }
+
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java:251]: Hi @jiancchen seems missing call updateCoppaStatus() method in the requestInterstitialAd().

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java [hunk 2]
```diff
@@ -474,6 +478,8 @@ public String getType() {
   public void loadRtbRewardedAd(@NonNull MediationRewardedAdConfiguration mediationRewardedAdConfiguration,
       @NonNull MediationAdLoadCallback<MediationRewardedAd, MediationRewardedAdCallback> mediationAdLoadCallback) {
     Log.d(TAG, "loadRtbRewardedAd()...");
+    VungleInitializer.getInstance()
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java:481]: We can remove this line due to we have done this in loadRewardedAd() method.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java (hunk 1 above)
Comment (jiancchen) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java:251]: @HuFangshuai agreed will need to update

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java (hunk 2 above)
Comment (jiancchen) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java:481]: @HuFangshuai thanks will verify

**Review by HuFangshuai — APPROVED** [review]
lgtm

**Comment by HuFangshuai** [comment]
Hi @jiancchen I have merged this PR with the master branch and feel free to merge this PR and create the upstream PR.

**Pull request merged**